README.md

```diff
@@ -13,60 +13,9 @@
 
 </div>
 
-## Installation
-
-Install the components that you need from npm:
-
-```sh
-npm install @vaadin/grid
-```
-
-## Usage
```

Comment:
Just to confirm, was the removal of this _entire_ section intentional?

Reply:
Yes, I feel like we don't need to have an extensive demo for a specific component in the main README.